src/main/webapp/script.js

```diff
@@ -211,3 +214,34 @@ for (let button of filterButtons) {
       }
     });
 }
+
+let infowindow;
+function updateRestaurantInfo(restaurant, marker) {
+  if (!infowindow) {
+    infowindow = new google.maps.InfoWindow();
+  }
+  let { geometry: {location}, placeId, name, photos, rating } = restaurant;
+
+  let params = {
+    api: 1,
+    destination: name,
+    destination_place_id: placeId
+  }
+  const dir_url = new URL('https://www.google.com/maps/dir/');
+  Object.keys(params).forEach(key => dir_url.searchParams.append(key, params[key]));
+
+  let contentString =
+    `<div id="info-container">
+      <h1 id="info-name">${name}</h1>
```

Comment:
Shouldnt use h1 for the restaurant name, its usually used for website title. Can use div instead and add some css styling.